### PR TITLE
Add `nodetimeline` command

### PR DESCRIPTION
**Pull request by bnb**
This PR is a first pass at a `nodetimeline` command. Before writing docs I'd like to get feedback on the API 🙌

Two notes:

I initially had an `--eol` flag but it did not play well with line 67, which checks what the latest version is – it seems `getEOL()` doesn't return identical data to the others (which is fine!).

I could also add checking a specific version, but figured I'd get this up since it's starting to get a bit late and I've been working on this for a few hours now.

### CODE REVIEW

File: bin/nodetimeline.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    icon = '☠️'
+  }
+
+  var stats = release.getStats()
+
+  console.log(icon + '  Node.js ' + release.version + ' Timeline')
+  console.log('      - Node.js ' + release.version + ' will be EOL in ' + stats.days.until.eol + ' days.')
+  console.log('      - Node.js ' + release.version + ' is ' + stats.percent.total + '% through its total lifespan.')
+  console.log('')
+  console.log('   ℹ️  Release Line Information')
```
Comment (bnb) [inline bin/nodetimeline.js]: Currently this doesn't print anything out on Node.js 10.11.0 😢

Comment (bnb) [reply bin/nodetimeline.js]: Resolved with https://github.com/nw/node-release-lines/pull/8/commits/2ae90c107a6038abc46a2e0b55097758653aedf0

**Comment by bnb** [comment]
Interestingly enough Node.js v10 is now returning `isActive` as `true`. Not sure if this is an issue or not 🤔

**Comment by nw** [comment]
> Interestingly enough Node.js v10 is now returning `isActive` as `true`, contrary to what the [README.md shows](https://github.com/nw/node-release-lines/blame/master/README.md#L89) (and what should _still_ be returned). Not sure if this is an issue or not 🤔

Two issues at play here:
* The schedule.json in nodejs/release repo indicates v10 goes into active LTS on Oct 1st (monday).
* There appears to be a slight rounding error with dates. It rounds to the closest day. Which for most purposes is fine, it just gets awkward in situations like this.

**Comment by nw** [comment]
Fun little tool. LGTM👍

Let's wait to see what happens tomorrow before we publish a release. If it does I need to start tracking v11.

**Comment by bnb** [comment]
Created an issue to check in on v10: https://github.com/nodejs/Release/issues/361

I’ll shoot over a PR including docs for `nodetimeline` later today 👍

**Pull request merged**